libraries/provider_runit_service.rb

```diff
@@ -304,12 +304,12 @@ def whyrun_supported?
 
         # Support supervisor owner and groups http://smarden.org/runit/faq.html#user
         if new_resource.supervisor_owner || new_resource.supervisor_group
-          directory "#{service_dir_name}/supervise" do
+          directory ::File.join(service_dir_name, supervise) do
```

Comment:
supervise here should be a string not a variable. That's causing failures in the tests